server/repo/repo-ui/src/main/java/org/eclipse/vorto/repository/web/ModelGenerationController.java

```diff
@@ -48,7 +48,7 @@
 /**
  * @author Alexander Edelmann - Robert Bosch (SEA) Pte. Ltd.
  */
-@Api(value="/generate", description="Generate Code from an Information Model")
```

Comment:
Here the idea would be to have a consistent description containing 'Inforamtion Model' such as:
Generate Code from an Information Model
Find an Information Model
Share an Information Model